backend/Cargo.toml

```diff
@@ -12,6 +12,7 @@ pil_analyzer = { path = "../pil_analyzer" }
 number = { path = "../number" }
 strum = { version = "0.24.1", features = ["derive"] }
 ast = { version = "0.1.0", path = "../ast" }
+pilopt = { path = "../pilopt" }
```

Comment:
```suggestion
```

I don't think this is needed?

Reply:
It is used here: https://github.com/powdr-labs/powdr/pull/656/files#diff-a26911fade45b717c44af0791ff3b5eaf89abe184928f407a419f7c3bb5c694aR388